src/dbnode/storage/types.go

```diff
@@ -1243,3 +1269,11 @@ type newFSMergeWithMemFn func(
 	dirtySeries *dirtySeriesMap,
 	dirtySeriesToWrite map[xtime.UnixNano]*idList,
 ) fs.MergeWith
+
+type AggregateTilesOptions struct {
+	Start, End          time.Time
+	Step                time.Duration
```

Comment:
nit: add comments on `Options` and on the variables

Reply:
ok